server/config/config.go

```diff
@@ -241,6 +256,7 @@ type Config struct {
 	Rosetta   RosettaConfig    `mapstructure:"rosetta"`
 	GRPCWeb   GRPCWebConfig    `mapstructure:"grpc-web"`
 	StateSync StateSyncConfig  `mapstructure:"state-sync"`
+	Streaming StreamingConfig  `mapstructure:"streamers"`
```

Comment:
```suggestion
	Streaming StreamingConfig  `mapstructure:"streaming"`
```

because
```
StreamingTomlKey = "streaming"
```
and
```
# Streaming allows nodes to stream state to external systems
[streaming]
```

Reply:
Yes, `streamers` is there to support in-process streaming. It's being used by crypto.org at the moment. It was requested and added a few months back as the ADR for the plugin system was in place. The `FileStreaming` has been removed because you can use the `ABCIListener` directly now but the configuration may still be need as it has a different structure and settings so I left it alone. I haven't changed this in upstream either. We can go either way on this but I felt it was best to leave it in until a change came from upstream.

Reply:
You've named it `"streamers"` here, but call it `"streaming"` everywhere else.

If you had a unit test that sets the config to something other than the defaults, you'd see that the streaming config isn't properly read into the config struct because it's looking for a "streamers" section.

Reply:
Hmm, that's a weird edge case. I'll take a deeper look. At first glance It looks like it should have been caught here.

https://github.com/provenance-io/cosmos-sdk/blob/9777eb95323564735eb209acf52fd84b94d20547/server/config/config_test.go#L68-L69

Reply:
It's not an edge case. It's named wrong. Viper cannot properly unmarshal it. The config file (and everywhere else you try to read it) calls the section "streaming" but you tell viper to read it from the "streamers" section when unmarshalling it.